src/content/faq/General.md

```diff
@@ -9,70 +9,71 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
 
 ## Is GraphQL a database language like SQL?
 
-<!-- TODO -->
+No, but this is a common misconception. GraphQL is designed for APIs, not databases. It’s a specification for how to request and modify data through an API layer. Then it determines how that data is communicated between servers and clients.
+
+GraphQL is frequently used for database interactions though. For example, it can be built into a wrapper around a database or a [GraphQL server](https://www.howtographql.com/advanced/1-server/) can query various databases. There are also [services](/code/#services) that implement the GraphQL specification and then allow clients to use GraphQL to query that data source. But it can also be used without a database at all.
 
-Sugar plum gummies donut cheesecake marshmallow donut caramels. Chocolate cake jelly beans ice cream halvah topping lemon drops liquorice croissant. Jelly-o chocolate gummies chocolate I love toffee danish sweet roll. Marzipan I love I love jujubes cupcake bear claw. Soufflé caramels caramels marzipan icing macaroon. Tootsie roll I love jelly beans halvah sweet donut fruitcake. Icing gummi bears tart marshmallow biscuit candy. Candy canes cotton candy soufflé muffin chocolate cake sweet. Candy canes dragée tiramisu gummies danish gummies gingerbread sugar plum gummies.
+For more information on how to get GraphQL to interact with your database, check out our [documentation on resolvers](/learn/execution/#root-fields-resolvers).
 
 ## Does GraphQL replace REST?
 
-<!-- TODO -->
+No, not necessarily. They both handle APIs and [serve the same purpose](/learn/thinking-in-graphs/#business-logic-layer) from a business perspective. GraphQL is considered an alternative to REST, but it’s not a definitive replacement.
 
-Jelly-o cupcake marshmallow marshmallow. Lemon drops cake caramels donut halvah cupcake topping tootsie roll. Jelly cotton candy candy cheesecake danish [jelly-o icing bonbon]().
+GraphQL and REST can actually co-exist in your stack. For example, you can abstract REST APIs behind a [GraphQL server](https://www.howtographql.com/advanced/1-server/). This can be done by masking your REST endpoint into a GraphQL endpoint using [root resolvers](/learn/execution/#root-fields-resolvers). 
 
-## How can I learn GraphQL?
+For an opinionated perspective on how GraphQL compares to REST, check out [How To GraphQL](https://www.howtographql.com/basics/1-graphql-is-the-better-rest/).
 
-<!-- TODO -->
+## How can I learn GraphQL?
```

Comment:
@IvanGoncharov Should we maybe recommend becoming familiar with the spec and SDL first? Or offer any sort of guidance for where new learners should start? Something more opinionated could work well for this answer.

Reply:
I know you didn't ask for my opinion but I'll write it anyway... :)

I believe the `learn` section, including everything in it is very important for people to follow. 
I see a lot of developers that skip through it directly to tutorials and they miss a lot of valuable information.
That includes the best practices sections as well.
After I'm directing them there they have a lot of AHA moments.

I don't personally like that we direct people at the top of the `learn` section to the tutorials, especially the 2 tutorials you are mentioning in the comment here.

Both of them are outdated and if people skip the `learn` section for them it makes for not a great starting experience.

As we should probably work and improve the tutorials section as well, I would recommend to direct people to the tutorial section, not mention specific tutorials and  then tackle the tutorial section to include the best tutorials first.
Hopefully that would lead to a better starting experience and also for people not to skip the important content on the `learn` section

Just my 2 cents

Reply:
I'm not sure what you mean by "tutorials section"... do you mean the best practices articles? Just want to make sure we're on the same page before I reply fully 😅 

So it's funny because I generally have the opposite opinion to you. Based on my conversations with new GraphQL learners, many come to the `Learn` section and immediately become overwhelmed. There's _so_ much information and concepts being thrown at them and realistically, you don't need to know all of these concepts to be productive with GraphQL. But you're right in that once you dive deeper into GraphQL and run into more use cases, knowing these can certainly help. 

Maybe we can find a compromise. Offer more 'quick' and practical materials for those who just want to be productive enough and then say that to understand the concepts more fully to work through the `Learn` section? 

I know a balance exists, but no one has seemed to find it yet aka why GraphQL is notoriously difficult to learn 😂

Reply:
By tutorials section I've meant something that might not exists yet 😂
I thought that in the future code or community sections we should have an updated list of all good and up to date tutorials.

And I can related to your opinion, I think they both are true (someone reads too much, then goes to tutorials, then forget to come back to read and misses things)

I think what you have right now is a great balance!
And in the future, once we'll get a good tutorial list, we could rethink it?